### PR TITLE
[Gbp-2179] Open Banking Docs

### PR DESCRIPTION
**Pull request by ttomttom**

### CODE REVIEW

File: source/includes/_openbanking.md [hunk 1]
```diff
@@ -0,0 +1,374 @@
...
+
+For a detailed description of requirements and supported algorithms please check out the Well-Known Open Banking URL.
+
+## Sandbox Access
+
+We highly recommend that you get started with connecting to our sandbox first, before moving on to production. To get started with the registration process drop us an email to `openbanking@transferwise.com`
+
+### Open Banking Directory
+
+If you're a registered TPP in the Open Banking Directory and you are using Signing and Transport certificates issue by
```
Comment (JamesFerretti) [inline source/includes/_openbanking.md]: Issued*

File: source/includes/_openbanking.md [hunk 2]
```diff
@@ -0,0 +1,374 @@
...
+Open Banking, you can easily onboard by providing either of the following things:
+
+* You can generate and send us a Software Statement Assertion (SSA) from Open Banking Directory.
```
Comment (JamesFerretti) [inline source/includes/_openbanking.md]: from the*

File: source/includes/_openbanking.md [hunk 3]
```diff
@@ -0,0 +1,374 @@
...
+
+We highly recommend that you get started with connecting to our sandbox first, before moving on to production. To get started with the registration process drop us an email to `openbanking@transferwise.com`
+
+### Open Banking Directory
+
+If you're a registered TPP in the Open Banking Directory and you are using Signing and Transport certificates issue by
+Open Banking, you can easily onboard by providing either of the following things:
+
+* You can generate and send us a Software Statement Assertion (SSA) from Open Banking Directory.  
+* Alternatively, you can also just sent us your Open Banking Organization ID together with the Software Statement ID that you wish to use.
```
Comment (JamesFerretti) [inline source/includes/_openbanking.md]: just send*

File: source/includes/_openbanking.md [hunk 4]
```diff
@@ -0,0 +1,374 @@
...
+
+In either case, please make sure to have your list of **Redirect Uri**-s correctly configured in your Software Statement, otherwise you'll not be able to go through to whole flow.
```
Comment (JamesFerretti) [inline source/includes/_openbanking.md]: the whole flow.*

File: source/includes/_openbanking.md [hunk 5]
```diff
@@ -0,0 +1,374 @@
...
+### eIDAS Certificates
+
+In case you're using an eIDAS certificates instead of the ones issued by the Open Banking Directory just contact us to 
+see how we can move forward. 
+
+## Sandbox Test User
+
+You can sign up for a test user account here [https://sandbox.transferwise.tech/register](https://sandbox.transferwise.tech/register). The 2FA code used for any subsequent logins to the sandbox will alway be `111111`.
+
+You'll be asked to set up a developer account by filling in your profile information. Once this is done, you'll see that your newly created profile comes with a couple of test accounts opened and some test balance in them.
```
Comment (JamesFerretti) [inline source/includes/_openbanking.md]: some test funds in them* perhaps?

File: source/includes/_openbanking.md [hunk 6]
```diff
@@ -0,0 +1,374 @@
...
+nonce          | Optional    | If present, it must also be present in the JWT
+
+You can take a look at the example JWT request object on the right. Please note that:
+
+* Every request parameter sent in the URL **must** also be present in the signed JWT.
+* The `aud` claim should be correctly filled as `https://openbanking.sandbox.transferwise.tech`
+* The `openbanking_intent_id` **must** be filled with a valid `ConsentId` issued to the TPP.
+* The JWT **must** be signed with a valid signing certificate.
+
+Once the TPP redirects the user browser to the Authorization Endpoint the next things will happen in order:
```
Comment (JamesFerretti) [inline source/includes/_openbanking.md]: the following things*

File: source/includes/_openbanking.md [hunk 7]
```diff
@@ -0,0 +1,374 @@
...
+
+* Being a registered TPP under the Open Banking Directory will ease the integration process.
+* We are using [mutualTLS](https://en.wikipedia.org/wiki/Mutual_authentication) as the means for authentication. Moreover, the `CN` of your client certificate is expected to match the `clientId` under which you are registered. 
+* You need to register your client, before you'll be able to call any of the APIs. Check out the [Sandbox Access](#open-banking-api-sandbox-access) first.
+
+For a detailed description of requirements and supported algorithms please check out the Well-Known Open Banking URL.
+
+## Sandbox Access
+
+We highly recommend that you get started with connecting to our sandbox first, before moving on to production. To get started with the registration process drop us an email to `openbanking@transferwise.com`
```
Comment (ilhantw) [inline source/includes/_openbanking.md]: to [openbanking@transferwise.com](mailto:openbanking@transferwise.com) ?

File: source/includes/_openbanking.md [hunk 8]
```diff
@@ -0,0 +1,374 @@
...
+
+* You can generate and send us a Software Statement Assertion (SSA) from Open Banking Directory.  
+* Alternatively, you can also just sent us your Open Banking Organization ID together with the Software Statement ID that you wish to use.
+
+In either case, please make sure to have your list of **Redirect Uri**-s correctly configured in your Software Statement, otherwise you'll not be able to go through to whole flow.
+
+### eIDAS Certificates
+
+In case you're using an eIDAS certificates instead of the ones issued by the Open Banking Directory just contact us to 
+see how we can move forward.
```
Comment (ilhantw) [inline source/includes/_openbanking.md]: would it make sense to add how to contact? I assume same openbanking@transferwise.com?

File: source/includes/_openbanking.md (hunk 7 above)
Comment (ttomttom) [reply source/includes/_openbanking.md]: Yup. The mailing list already exists. Makes sense to keep contacts separate from other public API related queries.

Comment (ilhantw) [reply source/includes/_openbanking.md]: Ok, github markdown eat my point. My point was use link annotation and mailto: form for improved usability.

**Pull request merged**